src/utils/parsers/react/index.ts

```diff
@@ -3,10 +3,18 @@ import { BaseParser } from '../BaseParser'
 const variableNameCapturePattern = /([^,)]*)/
 const defaultValueCapturePattern = /(?:[^),]*|{[^}]*})/
 
+const supportedMethods = [
+    'useDVCVariable',
```

Comment:
this one is already deprecated..still worth adding?

Reply:
If we're sure no one is using an old version of the sdk — Mark was testing these commands and his codebase was still using `useDVCVariable`

Reply:
So I would say it doesn't hurt to have it for now. It's very easy to remove it later